var/spack/repos/builtin/packages/mvapich/package.py

```diff
@@ -59,11 +59,18 @@ class Mvapich(AutotoolsPackage):
         multi=False,
         description="Number of bits allocated to the rank field (16 or 32)",
     )
+    variant(
+        "pmi_version",
+        description="Which pmi version to be used. If using pmi2 add it to your CFLAGS",
+        default="simple",
+        values=("simple", "pmi2"),
+        multi=False,
+    )
 
     variant(
         "process_managers",
         description="List of the process managers to activate",
-        values=disjoint_sets(("auto",), ("slurm",), ("hydra", "gforker", "remshell"))
+        values=disjoint_sets(("auto",), ("None",), ("slurm",), ("hydra", "gforker", "remshell"))
```

Comment:
Please don't. If you want to have an identifier for an empty set, just remove the line below:
```python
.phoibit_empy_set()
```